TicTacToe/Constants.cs

```diff
@@ -0,0 +1,28 @@
+
+namespace TicTacToe
+{
+    public static class Constants
+    {
+        public const int EmptyCellValue = 0;
+        public const string EmptyCellMark = ".";
+        
+        public const int PlayerXValue = 1;
```

Comment:
I'm wondering if we have over complicated things a bit by keeping the board as an int[,]. Maybe it would be simpler if it was a string[,]. That way we wouldn't have to keep track of PlayerXValue and PlayerXMark.